stubs-common-aarch32/stubs-common-aarch32.cabal

```diff
@@ -0,0 +1,53 @@
+cabal-version:      2.4
+name:               stubs-common-aarch32
+version:            0.1.0.0
+
+-- A short (one-line) description of the package.
+-- synopsis:
+
+-- A longer description of the package.
+-- description:
+
+bug-reports: https://github.com/GaloisInc/stubs/issues
+license: BSD-3-Clause
+author: Christian Fontenot
+maintainer: dmatichuk@galois.com, langston@galois.com, rscott@galois.com
+copyright: (c) Galois, Inc 2023-2025
+
+source-repository head
+  type: git
+  location: https://github.com/GaloisInc/stubs
+  subdir: stubs-common-aarch32
+
+-- category:
```

Comment:
Remove? (Same comment for other `.cabal` files)

Reply:
Yeah, this is a defect that pretty much all of the `.cabal` files in this repo share (not just the ones that I added). I'll push a separate commit to fix all occurrences of this issue.